### PR TITLE
Refactor dqn word choice

### PR DESCRIPTION
**Pull request by vwxyzjn**
## Description
This PR fixes a confusing word choice in our DQN implementation. Previously we had

`logits = q_network.apply(q_state.params, obs)`

However, the word logits usually refer to "unnormalized probabilities" (see [What is the meaning of the word logits in TensorFlow?](https://stackoverflow.com/questions/41455101/what-is-the-meaning-of-the-word-logits-in-tensorflow)). A more correct word of choice here is `q_values = q_network.apply(q_state.params, obs)`.

Given this is a **non-performance-impacting** refactor, re-running the benchmark is not required.

CC @santiontanon

## Types of changes

- [ ] Bug fix
- [ ] New feature
- [ ] New algorithm
- [x] Documentation

### CODE REVIEW

**Comment by vwxyzjn** [comment]
@yooceii could you approve this please?

**Review by yooceii — APPROVED** [review]
